0.10/test/test-app/README.md

```diff
@@ -2,3 +2,10 @@ node-echo
 =========
 
 node.js echo server, returns request data to response
+
+## Using existing build tools
+
+### Integrating s2i-nodejs developer support with Grunt and Gulp
+You can use Grunt or Gulp to detect changes in your local directory and sync those changes to the code running in the remote docker container.
+_Gruntfile.js_ and _gulpfile.js_ provide apt examples for doing this. 
```

Comment:
I can remove the docs here and put it into a blog that's tailored towards `oc` use. @gorkem what do you think?